docs/content/querying/querying.md

```diff
@@ -122,4 +122,5 @@ Possible codes for the *error* field include:
 |`Query cancelled`|The query was cancelled through the query cancellation API.|
 |`Resource limit exceeded`|The query exceeded a configured resource limit (e.g. groupBy maxResults).|
 |`Unauthorized request.`|The query was denied due to security policy. Either the user was not recognized, or the user was recognized but does not have access to the requested resource.|
+|`Unsupported operation`|The query attempted to perform an unsupported operation|
```

Comment:
> The query attempted to perform an unsupported operation. This may occur when using undocumented features or when using an incompletely implemented extension.